daemon/containerio/cio.go

```diff
@@ -198,6 +198,10 @@ func WithAttach(stdin io.Reader, stdout, stderr io.Writer) containerdio.Attach {
 		if paths == nil {
 			return nil, fmt.Errorf("cannot attach to existing fifos")
 		}
+		// judge stdin is initial, make logic same with copyIO.
+		if i, ok := stdin.(*ContainerIO); (ok && i == nil) || (stdin == nil) {
+			paths.In = ""
+		}
```

Comment:
I just wonder if this code is need here, code not so good.

Reply:
@Ace-Tang What did you concern about? Maybe we should use `reflect.ValueOf(stdin).IsNil()` to check whether stdin is nil?